.github/dependabot.yaml

```diff
@@ -2,6 +2,19 @@ version: 2
 updates:
   - package-ecosystem: "docker"
     directory: "/"
+    labels:
+      - "dependencies"
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "gomod"
+    directory: "/"
+    labels:
+      - "dependencies"
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    labels:
+      - "dependencies"
     schedule:
       interval: "daily"
-
```

Comment:
Note: always keep a new line at the end of files for Unix terminal and editor compatibility/convenience.